lib/Dialect/SV/SVOps.cpp

```diff
@@ -322,6 +366,23 @@ static void printEventList(OpAsmPrinter &p, AlwaysOp op, ArrayAttr portsAttr,
   }
 }
 
+void AlwaysOp::getCanonicalizationPatterns(OwningRewritePatternList &results,
+                                           MLIRContext *context) {
+  // If the body is empty, erase op.
+  struct EraseEmptyOp final : public OpRewritePattern<AlwaysOp> {
+    using OpRewritePattern::OpRewritePattern;
+    LogicalResult matchAndRewrite(AlwaysOp op,
+                                  PatternRewriter &rewriter) const override {
+      if (!op.body().empty())
```

Comment:
Should this use isEmptyBlockExceptForTerminator?  It has a terminator too IIRC.

Reply:
It looks `AlwaysOp` and `InitialOp` have been deleted without this PR so I have removed it